drivers/video/video_mcux_csi.c

```diff
@@ -126,7 +126,6 @@ static int video_mcux_csi_set_fmt(const struct device *dev, enum video_endpoint_
 		return -EINVAL;
 	}
 
-	data->pixelformat = fmt->pixelformat;
```

Comment:
Can we drop the `pixelformat` field from the `data` structure as well?

Reply:
yes, I forgot this, thanks